README.md

```diff
@@ -253,6 +253,7 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [Weekly Python Newsletter](https://importpython.com/newsletter/). Weekly Python Newsletter containing Python Articles, Projects, Videos, Tweets delivered in your inbox. [Archive](https://importpython.com/newsletter/archive/).
 - [Pycoders Weekly](https://pycoders.com/). A free weekly e-mail newsletter, on Fridays, for those interested in python development and various topics around python.
 - [Awesome Python Newsletter](https://python.libhunt.com/newsletter). A weekly overview of the most popular Python news, articles and packages.
+- [Data Science Simplified](https://mathdatasimplified.com/). A daily Python and data science snippet
```

Comment:
could you fix the format? `.` at the end.

Reply:
I have just added it. Let me know if it looks good to you

Reply:
sure, thanks